lib/JSON/Validator/OpenAPI/Mojolicious.pm

```diff
@@ -19,6 +19,28 @@ has version => 2;
 
 sub E { JSON::Validator::Error->new(@_) }
 
+has generate_definitions_path => sub {
+  my $self = shift;
+  Scalar::Util::weaken($self);
+
+  return sub {
+    my $ref = shift;
+    my @path;
+
+    # Try to determin the path from the fqn
```

Comment:
s/determin/determine

Reply:
Could you also add a comment with an example fqn? It's a bit hard to me from the code to guess what goes into `$relative` and `$path`.